src/test/java/seedu/address/logic/commands/GetMonthlyFeeCommandTest.java

```diff
@@ -22,7 +23,7 @@
     // Alice has 4 session and has only 2 session that is on 1 Jan 2021
     // The other 2 session, one has a different month, one has a different year
     @Test
-    public void execute_correctMonthlyFee_success() throws CommandException {
+    public void execute_correctMonthlyFee_success1() throws CommandException {
```

Comment:
Actually now that I think about this again, maybe we should build a new temporary student from scratch, add it to the list of students temporarily, then feed it through `GetMonthlyFeeCommand` and compare the results with a 'hardcoded' result. Not too sure how it'll improve efficiency though, but just a suggestion! shouldn't be an issue if we continue with this current implementation.

Reply:
O.o hmmmm, we sort of built the ALICE student from scratch also in typical students tou. I am not sure if creating another new student at this point for calculation would have any difference from using existing students for calculation. In regards to hardcoding the answer to check, I am not sure but then I don't really see hardcoded values in test case also, so not sure if its the correct way to do it ah. Thats y eventually I changed to this that seems to follow what the other test cases in AB3 do ah.

Reply:
was thinking we can build a new student so we can hardcode the results HAHA since ALICE might be susceptible to edits in future. but it's ok, let's keep it this way!!